README.md

```diff
@@ -173,6 +173,95 @@ crud.insert_object('customers', {
 ...
 ```
 
+### Insert many
```

Comment:
Nit: Don't forget to update the table of contents (use the `doctoc` tool).

Reply:
Done.